services/postprocessing/README.md

```diff
@@ -0,0 +1,31 @@
+# Postprocessing Service
+
+The `postprocessing` service handles the coordination of asynchronous postprocessing steps. 
+
+## General Prerequisites
+
+To use the postprocessing service, an event system needs to be configured for all services. By default, `ocis` ships with a preconfigured `nats` service.
+
+## Postprocessing Functionality
+
+The storageprovider service (`storage-users`) can be configured to initiate asynchronous postprocessing by setting the `STORAGE_USERS_OCIS_ASYNC_UPLOADS` environment variable to `true`. If this is the case, postprocessing will get initiated *after* uploading a file and all bytes have been received.
+
+The `postprocessing` service will then coordinate configured postprocessing steps like scanning the file for viruses. During postprocessing, the file will be in a `processing state` where only a limited set of actions are available. Note that this processing state excludes file accessability by users.
+
+When all postprocessing steps have completed successfully, the file will be made accessible for users.
+
+## Additional Prerequisites for the `postprocessing` Service
+
+When postprocessing has been enabled, configuring any postprocessing step will require the requested services to be enabled and pre-configured. For example, to use the `virusscan` step, one needs to have an enabled and configured `antivirus` service. 
+
+## Postprocessing Steps
+
+As of now, `ocis` allows two different postprocessing steps to be enabled via an environment variable.
+
+### Virus Scanning
+
+To enable virus scanning as a postprocessing step after uploading a file, the environment variable  `POSTPROCESSING_VIRUSSCAN` needs to be set to ` true`. As a result, each uploaded file gets virus scanned as part of the postprocessing steps. Note that the `antivirus` service is required to be enabled and configured for this to work.
+
+### Delay
+
+Though this is for development purposes only and NOT RECOMMENDED on production systems, setting the environment variable `POSTPROCESSING_DELAY` to a duration not equal to zero will add a delay step with the configured amount of time. ocis will continue postprocessing the file after the configured delay.
```

Comment:
Do we want to say what format the setting takes? A simple integer that is interpreted as seconds, can it take values like `1m` (one minute) `5s` (5 seconds. I don't know, and haven't dug around to see where the format might be documented.\
If documented somewhere else, then we could link to that.

Reply:
The format is a "duration" like 2s or 1m etc. It will be documented in the admin documentation. When this text is merged, I will transport it to the new postprocessing page which is in preperation and will include all values and its descriptions in detail.